Update the error burst alert runbooks (#229)
kubevirt/monitoring: docs/runbooks/VirtApiRESTErrorsBurst.md

# VirtApiRESTErrorsBurst

## Meaning

> For the last 10 minutes or longer, over 80% of the REST calls made to `virt-api` pods have consistently failed.

Reviewer: - Is "consistently" referring to "over 80%"? I think "over 80%" is sufficient unless "consistently" is referring to something else.
- Does the alert fire every 10 minutes? If so, I don't think we need to specify "for the last 10 minutes or longer." While that might be literally true, it makes more sense to me to limit the message to the actual scope of the alert.

Would the following phrasing be sufficient? 

```suggestion
During the past 10 minutes, more than 80% of REST calls made to `virt-api` pods have failed.
```

Author: @ousleyp We calculate the % of failed  REST calls in the last 5 minutes and if we see that it stays over 80% for additional 5 minutes, the alert will start firing. 
So in Total we can say we have the issue for 10 minutes or more.

The alert will continue firing until the issue is fixed and the % of the  REST calls in the last 5 minutes will be below 80%.

It means there is a very serious issue and most of the  REST calls fail.
WDYT?

> For at least the last 10 minutes, more than 80% of the REST calls to `virt-api` pods have been failing consistently.

Reviewer 2: nit: `For at least the last 10 minutes` -> `For the past/last 10 minutes or longer, ...`

Author: @assafad  Updated
